data/iso4217/amendments/168.json

```diff
@@ -0,0 +1,628 @@
+[
+  {
+    "code": "AED",
+    "base": 10,
+    "exponent": 2,
+    "description": "United Arab Emirates dirham"
+  },
+  { "code": "AFN", "base": 10, "exponent": 2, "description": "Afghan afghani" },
+  { "code": "ALL", "base": 10, "exponent": 2, "description": "Albanian lek" },
+  { "code": "AMD", "base": 10, "exponent": 2, "description": "Armenian dram" },
```

Comment:
@sarahdayan I would prefer them to all be formatted the same, would it be acceptable to ignore this file in prettier?